DashAI/back/dataloaders/params_schemas/csvdataloader.json

```diff
@@ -4,6 +4,17 @@
     "description": "Upload a .csv file with your dataset or a .zip file with CSV files to be uploaded.",
     "display": "modal",
     "properties": {
+        "name": {
+            "oneOf": [
+                {
+                    "error_msg": "",
+                    "description": "<p style='font-size: 15px;'>Dataset name</p><p style='font-size: 12px;'>If left <b>blank</b>, the name of the uploaded file will be used.<p/><a style='color: #ADD8E6;' href='https://github.com/DashAISoftware/DashAI'>More info.</a>",
```

Comment:
Not for this pr ;)

Reply:
Ok!, I did it with HTML initially to avoid installing another library, but I also realized that it's difficult to write the descriptions in HTML.
I agree that Markdown would be a better option.